DigitalAssetManagement/ServiceHelpers/TelemetryHelper.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace ServiceHelpers
+{
+    public static class TelemetryHelper
```

Comment:
Let's call this ErrorTrackingHelper to be consistent with how we are doing it in other samples: https://github.com/microsoft/Cognitive-Samples-IntelligentKiosk/blob/master/Kiosk/ServiceHelpers/ErrorTrackingHelper.cs